toolchain-msvc.cmake

```diff
@@ -39,6 +39,8 @@ if(USE_CLANG_CL)
         set(CMAKE_C_COMPILER_TARGET "arm64-pc-windows-msvc")
         set(CMAKE_CXX_COMPILER_TARGET "arm64-pc-windows-msvc")
     else()
+        set(CMAKE_C_FLAGS "-m32")
+        set(CMAKE_CXX_FLAGS "-m32")
```

Comment:
Add a comment like '`-m32` is needed when using mixed (Clang-CL x64 with MSVC x86) tools'?

Reply:
A comment wound be nice, yes.

Reply:
Done.